zerver/migrations/0001_initial.py

```diff
@@ -412,11 +401,6 @@ class Migration(migrations.Migration):
             field=models.DateTimeField(auto_now=True),
         ),
         migrations.AlterField(
-            model_name='referral',
-            name='email',
-            field=models.EmailField(max_length=254),
-        ),
-        migrations.AlterField(
```

Comment:
Don't edit this file: it's intended to be the initial version of our database schema.  Instead, you want to create a new migration with `manage.py makemigrations`.